tests/etl/test_perf_data_load.py

```diff
@@ -0,0 +1,221 @@
+import copy
+import datetime
+import json
+import time
+
+import pytest
+
+from tests.etl.test_perf_data_adapters import _verify_signature
+from tests.test_utils import create_generic_job
+from treeherder.etl.perf import store_performance_artifact
+from treeherder.model.models import Push
+from treeherder.perf.models import (PerformanceDatum,
+                                    PerformanceFramework,
+                                    PerformanceSignature)
+
+FRAMEWORK_NAME = 'cheezburger'
+MEASUREMENT_UNIT = 'ms'
+UPDATED_MEASUREMENT_UNIT = 'seconds'
+
+
+@pytest.fixture
+def datum():
```

Comment:
This name is too generic. Who's outputting this kind of data in our pipelines?
We need to find out & update this `datum` identifier so that it reflects more
about this type of data.

Reply:
Ok. Will use `sample_perf_artifact`.